### DISCUSSION TITLE
Show & Tell: Introducing patterns.run

### DISCUSSION THREAD

**Discussion by grikomsn** [codesandbox/sandpack#242] [Show and tell]
https://patterns.run is a collection of snippets to render logical patterns, remembering the terminal and C++ days of those assignments where you need to print a specific pattern using for loops of do whiles.

[<img width="1492" alt="image" src="https://user-images.githubusercontent.com/8220954/144918696-e2bfa988-3736-4e8c-8c69-fd6ea7f8027b.png">](https://patterns.run/pattern/arrow-bottom-left)

Previously it was made using [react-live](https://github.com/FormidableLabs/react-live) which at the time was an initial proof of concept and sub-par editing experience. Next iteration added [Monaco editor](https://github.com/suren-atoyan/monaco-react) but still relies on react-live to compute and render the pattern. With [Sandpack](https://sandpack.codesandbox.io/), now it handles both editing and pattern rendering!

Future roadmap will include playground to create your own pattern and submit to GitHub. Let me know what you think! 👋🏻

**Comment by danilowoz** [comment]
Hey, @grikomsn, that looks awesome! What a great project, I already spent a quiet time playing around with it.

I'm curious to hear from you if you have any feedback or anything unexpected you faced while using Sandpack? 
We're very focused on shipping a great experience for developers, and even those tiny details might be essential for us.
